scripts/start.js

```diff
@@ -4,6 +4,8 @@
 process.env.BABEL_ENV = process.env.BABEL_ENV || 'development';
 process.env.NODE_ENV = process.env.NODE_ENV || 'development';
 
+process.env.REACT_APP_VERSION = process.env.REACT_APP_VERSION || 'x.y.z';
```

Comment:
use `process.env.npm_package_version` which is a standard node env variable loaded from `package.json`

Reply:
This did not work, where did you find this information?